src/main/java/homework_4/components/Result.java

```diff
@@ -0,0 +1,112 @@
+package homework_4.components;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by dinar on 28.11.2019.
+ */
+public class Result {
+    private int summary;
+    private String color;
+    private String metals;
+    private List<String> vegetables;
+    private List<String> elements;
+
+    private Result(int summary, String color, String metals, List<String> vegetables, List<String> elements) {
+        this.summary = summary;
+        this.color = color;
+        this.metals = metals;
+        this.vegetables = vegetables;
+        this.elements = elements;
+    }
+
+    public int getSummary() {
+        return summary;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public String getMetals() {
+        return metals;
+    }
+
+    public List<String> getVegetables() {
+        return vegetables;
+    }
+
+    public List<String> getElements() {
+        return elements;
+    }
+
+    public static ResultBuilder resultBuilder() {
+        return new ResultBuilder();
+    }
+
+    public static class ResultBuilder {
+        //default value, if change nothing in the page
+        private int summary = 3;
+        private String color = "Colors";
+        private String metals = "Metals";
+        private List<String> vegetables = new ArrayList<>();
+        private List<String> elements;
```

Comment:
Why do you decide add default values to the builder?

Reply:
Because Result always will be have it, even if we chose nothing, item Result will be include this values by default